src/css/styles.css

```diff
@@ -9,7 +9,7 @@ p {
 }
 
 h1 {
-  color: #E98A15;
+  color: #ffffff;
```

Comment:
I continue to not like this and wish there was a better way to match color schemes than the one in chrome's dev tools and probably there is but that is absolutely a tomorrow David problem.